charts/opentelemetry-operator/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: opentelemetry-operator
-version: 0.9.4
+version: 0.9.5
```

```suggestion
version: 0.10.0
```

We typically bump the minor version when updating the operator/collector versions